src/progress-steps/styled-components.js

```diff
@@ -36,25 +36,24 @@ export const StyledIcon = styled<StylePropsT>(
   ({$theme, $isActive, $isCompleted, $disabled}) => {
     let currentColor = $theme.colors.mono400;
     let size = $theme.sizing.scale300;
-    let marginRight = $theme.sizing.scale500;
-    let marginLeft = $theme.sizing.scale100;
+    let marginLeft = $theme.sizing.scale850;
```

Comment:
We should keep the margins unchanged here since this comment is already used in many applications. Updating this may change page alignment too much

Reply:
Yes, but on the design team's spec. A main task is to adjust all the margins. https://www.figma.com/file/WDDZWB60wmzbZtiSG53UDh/%E2%9D%96-Base-documentation
So just skip the margin change the design team want?

Reply:
I will find a way by reducing other margins to make the total width as close as the previous one.